Content.Shared/Weapons/Ranged/Components/AutoShootGunComponent.cs

```diff
@@ -21,12 +21,10 @@ public sealed partial class AutoShootGunComponent : Component
     /// <summary>
     /// Frontier - Whether or not the gun can actually fire (i.e. switched on and receiving power if needed)
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite)]
     public bool CanFire;
 
     /// <summary>
     /// Frontier - Amount of power this gun needs from an APC in Watts to function.
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite)]
     public float OriginalLoad { get; set; } = 0;
```

Comment:
You bring up a good point - should these fields only exist server-side?

Reply:
Also, it might be nice to have these variables as read-only in VV, if only to debug if/why a particular gun won't fire.